app/grandchallenge/documentation/admin.py

```diff
@@ -0,0 +1,25 @@
+from django.contrib import admin
+from django.forms import ModelForm
+from markdownx.admin import MarkdownxModelAdmin
+
+from grandchallenge.documentation.models import DocPage
+
+
+class AdminDocPageForm(ModelForm):
+    class Meta:
+        model = DocPage
+        exclude = ()

```

Comment:
Isn't this another way of doing `__all__`? If so, I would prefer if we are explicit about what fields to include, even for admins.